packages/legend-query-builder/src/stores/QueryBuilderStateBuilder.ts

```diff
@@ -740,16 +745,24 @@ export class QueryBuilderValueSpecificationProcessor
       );
       return;
     } else if (
-      matchFunctionName(
-        functionName,
+      matchFunctionName(functionName, [
         QUERY_BUILDER_SUPPORTED_FUNCTIONS.TDS_GROUP_BY,
-      )
+        QUERY_BUILDER_SUPPORTED_FUNCTIONS.RELATION_GROUP_BY,
+      ])
     ) {
-      processTDSGroupByExpression(
-        valueSpecification,
-        this.queryBuilderState,
-        this.parentLambda,
-      );
+      if (isTypedGroupByExpression(valueSpecification)) {
```

Comment:
same comment here, we should be careful of the case where people specify the fully specified function path